ci/build-all-steps.yml

```diff
@@ -62,9 +62,16 @@ steps:
   env:
     PACKAGE_TOKEN: $(Package.Token)
 
-# Build
+# Build BuildConfigGen
+- task: DotNetCoreCLI@2
+  displayName: Build BuildConfigGen
+  inputs:
+    command: build
+    projects: './BuildConfigGen/BuildConfigGen.csproj'
```

Comment:
Adding a task to build BuildConfigGen project in case when no tasks changes and only BuildConfigGen related changes are in the PR. This way we can make sure the project was built successfully.